### PR TITLE
add relay click driver

### PR DESCRIPTION
**Pull request by pratik-prajapati-imgtec**
Signed-off-by: Pratik Prajapati Pratik.Prajapati@imgtec.com

### CODE REVIEW

**Comment by nikhil-zinjurde-imgtec** [comment]
test with jenkins

**Comment by nikhil-zinjurde-imgtec** [comment]
test with jenkins

File: platform/mikro-e/dev/relay-click.c [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+{
+  switch(relay_num) {
+    case relay1:
+      GPIO_INV(RELAY1_PORT, RELAY1_PIN);
+      break;
+    case relay2:
+      GPIO_INV(RELAY2_PORT, RELAY2_PIN);
+      break;
+    default:
+      printf("Invalid relay to set off\n");
```
Comment (rahul-daga-imgtec) [inline platform/mikro-e/dev/relay-click.c]: Invalid relay to toggle

File: platform/mikro-e/dev/relay-click.h [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ *         Relay click header file.
+ *         http://www.mikroe.com/click/relay/
+ */
+
+#ifndef RELAY_CLICK_H_
+#define RELAY_CLICK_H_
+
+typedef enum
+{
+	relay1=1,
```
Comment (rahul-daga-imgtec) [inline platform/mikro-e/dev/relay-click.h]: Caps

Comment (rahul-daga-imgtec) [inline platform/mikro-e/dev/relay-click.h]: Verified

**Comment by rahul-daga-imgtec** [comment]
Approved

**Comment by rahul-daga-imgtec** [comment]
Verified

**Comment by abhijit-mahajani-imgtec** [comment]
Approved

**Comment by nikhil-zinjurde-imgtec** [comment]
Accepting "Verified by rahul-daga-imgtec"

**Pull request merged**